src/ForwardDiffNumber.jl

```diff
@@ -128,6 +131,9 @@ isfinite(n::ForwardDiffNumber) = isfinite(value(n))
 isinf(n::ForwardDiffNumber) = isinf(value(n))
 isreal(n::ForwardDiffNumber) = isconstant(n)
 
+floor{T}(::Type{T}, n::ForwardDiffNumber) = floor(T, value(n))
```

Comment:
What do you do with `n = ForwardDiff.GradientNumber(3,(1,-2))`? Is it less than 3 or bigger than 3?